apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -183,17 +190,43 @@ class CensusForm extends Component {
   }
 
   toggleTopics(option) {
-     const selected = this.state.selectedTopics.slice(0);
-     if (selected.includes(option)) {
-       this.setState({
-         selectedTopics: _.without(selected, option)
-       });
-     } else {
-       this.setState({
-         selectedTopics: selected.concat(option)
-       });
-     }
-   }
+    const selected = this.state.selectedTopics.slice(0);
+    if (selected.includes(option)) {
+      this.clearOption(option);
+    } else {
+      this.selectOption(option);
+    }
+  }
+
+  selectOption(option) {
+    const selected = this.state.selectedTopics.slice(0);
+    this.setState({
+      selectedTopics: selected.concat(option)
+    });
+  }
+
+  clearOption(option) {
+    const selected = this.state.selectedTopics.slice(0);
```

Comment:
Why do we need this `.slice(0)` repeated (lines 193, 202, 209), or even at all? 

If it was needed to copy the array for updating state, I'd suggest consolidating instead of doing it 3 times. However, it does not even appear to be needed for that. Both [Array.prototype.concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat?v=a) (line 204) and [Lodash.without](https://lodash.com/docs/4.17.4#without) (line 211) return copies of the array rather than mutating the input. I think all 3 of these identical `slice(0)` lines can be removed.

Reply:
I agree. I will clean this up.